promql/histogram_stats_iterator.go

```diff
@@ -26,20 +26,14 @@ import (
 type HistogramStatsIterator struct {
```

Comment:
Should we add a comment detail to mention it returns `ValFloatHistogram` only (the fact it treats inthistograms as float under the hood)?